### PR TITLE
WIP - EM-1026: added "not designated reviewable" to drop down box

### PR DESCRIPTION
**Pull request by kylecjo**
@esune @NickPhura

### CODE REVIEW

File: modules/projects/client/views/project-partials/project-tombstone.html [hunk 1]
```diff
@@ -48,7 +48,7 @@ <h2>Project Details</h2>
 			</div>
 			<div class="col-sm-4 col-md-4" ng-if="project.eacDecision">
 				<div class="detail-row">
-					<label>EAC Decision</label>
+					<label>EA Decision</label>
```
Comment (NickPhura) [inline modules/projects/client/views/project-partials/project-tombstone.html]: Should this actually be "EA" and not "EAC"?

Looks like all of the variables use "eac".

Comment (kylecjo) [reply modules/projects/client/views/project-partials/project-tombstone.html]: The ticket says to update the display label from "EAC Decision" to "EA Decision"

**Comment by esune** [comment]
Holding off from merging this until we clarify our plan to push the current changes to prod.

File: modules/projects/client/views/project-partials/project-tombstone.html (hunk 1 above)
Comment (NickPhura) [reply modules/projects/client/views/project-partials/project-tombstone.html]: Oh gotcha.  Thats fair then.

**Pull request merged**